lib/phoenix_ecto/plugs/check_repo_ready.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Phoenix.Ecto.Plugs.CheckRepoReady do
+  @moduledoc """
+  A plug that does some checks on your application repos.
+
+  Checks if the storage is up (database is created) or if there are any pending migrations.
+  Both checks can raise an error if the conditions are not met.
+
+  ## Plug options
+
+  * `:otp_app` - name of the application which the repos are fetched from
```

Comment:
Please indent bullets with two spaces.

Reply:
I always forget this, nice! 😄

[Fixed here](https://github.com/phoenixframework/phoenix_ecto/compare/1e27d0ed46a5039f4dbaa0f7503631898a453a77..846278a787c3300abfadce9a9d8097da9131afe2)